+++
title = "Polygenic risk score based on weight gain trajectories is a strong predictor of childhood obesity"
date = 2019-04-11T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["SJC Craig", "AM Kenney", "J Lin", "IM Paul", "LL Birch", "JS Savage", "ME Marini", "F Chiaromonte", "M Reimherr", "KD Makova"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "_bioRxiv_"
publication_short = "_BIORXIV_"

# Abstract and optional shortened version.
abstract = "Obesity is highly heritable, yet only a small fraction of its heritability has been attributed to specific genetic variants. Missing heritability is particularly pronounced for childhood obesity. Here we studied 226 children for whom we typed almost one million single-nucleotide polymorphisms (SNPs), and collected weight and length or height at eight time points between birth and the age of three years. Leveraging longitudinal weight gain trajectory information and novel functional data analysis (FDA) techniques, we constructed a polygenic risk score (PRS) comprised of 24 SNPs. This PRS explains 56% of the variability in weight gain trajectories among the studied children. Moreover, it is significantly higher in children with (vs. without) rapid infant weight gain—a predictor of obesity later in life. We validated the constructed PRS in populations of adolescents and adults—suggesting that some genetic variants predispose to obesity at both childhood and later life stages. In contrast, PRSs from genome-wide association studies (GWAS) of adult obesity were not predictive of weight gain in our cohort of children, and did not share SNPs with our PRS. Our research provides a strong example of a successful application of FDA to a GWAS. We demonstrate that a sophisticated characterization of a longitudinal phenotype can provide increased statistical power to studies with smaller sample sizes. This has the potential of shifting the existing paradigm in GWAS."
abstract_short = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["childhood_obesity"]


# Slides (optional).
#   Associate this page with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides = ""

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []

# Links (optional).
url_pdf = ""
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]


# Digital Object Identifier (DOI)
doi = "10.1101/606277"


# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""
+++
