+++
title = "Non-canonical DNA in human and other ape telomere-to-telomere genomes"
date = 2024-09-03T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["L Smeds", "K Kamali", "KD Makova"]


# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "_bioRxiv_"
publication_short = "_BIORXIV_"

# Abstract and optional shortened version.
abstract = "Non-canonical (non-B) DNA structures–bent DNA, slipped-strand structures, palindromes and hairpins, triplex DNA, G-quadruplexes, etc.–which form at certain sequence motifs (A-phased repeats, direct repeats, inverted repeats, mirror repeats, etc.), have recently emerged as important regulators of many cellular processes and as drivers of genome evolution. Yet, they have been understudied due to their highly repetitive nature and potentially inaccurate sequences when examined with short-read technologies. The recent availability of human and other ape telomere-to-telomere (T2T) genomes, which were generated with long reads, provides an opportunity to study the repertoire of non-canonical DNA motifs comprehensively. Here we characterized non-B DNA motifs in the T2T genomes of human and six non-human apes–bonobo, chimpanzee, gorilla, Bornean orangutan, Sumatran orangutan, and siamang. We found that non-B DNA motifs are enriched at the genomic regions that were added to these T2T assemblies as compared to previous assembly versions. Non-B DNA motifs occupied 9-15%, 9-11%, and 12-38% of autosomal, chromosome X, and chromosome Y sequences, respectively. Gorilla had the highest percentage of its genome occupied by non-B DNA motifs. The same DNA sequences were frequently annotated with multiple non-B DNA motif types. Several types of non-B DNA motifs had high densities at short arms of acrocentric chromosomes and non-canonical structures might contribute to satellite dynamics in these regions. Most centromeres showed an enrichment in at least one non-B DNA motif type, consistent with the role of non-B structures in determining centromeres. Our results highlight the uneven distribution of predicted non-B DNA structures across ape genomes and suggest their novel functions in the previously inaccessible genomic regions."
abstract_short = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["microsatellite_variation_and_evolution"]


# Slides (optional).
#   Associate this page with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides = ""

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []

# Links (optional).
url_pdf = ""
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]

# Digital Object Identifier (DOI)
doi = "https://doi.org/10.1101/2024.09.02.610891"

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""
+++
