+++
title = "Substitution spectrum and selection at G-quadruplexes in great ape telomere-to-telomere genomes"
date = 2026-02-17T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["X Zhang", "S Mohanty", "F Chiaromonte", "KD Makova"]

# Publication type.
# Legend:
# 0 = Uncategorized
# 1 = Conference paper
# 2 = Journal article
# 3 = Manuscript
# 4 = Report
# 5 = Book
# 6 = Book section
publication_types = ["2"]

# Publication name and optional abbreviated version.
publication = "_Genome Biology and Evolution_"
publication_short = "_GENOME BIOL EVOL, ACCEPTED_"

# Abstract and optional shortened version.
abstract = "G-quadruplexes (G4s) are noncanonical DNA secondary structures formed by runs of guanines (stems) connected by other nucleotides (loops). These structures are enriched at regulatory regions such as promoters, CpG islands, untranslated regions (UTRs), enhancers, and replication origins, where they play key roles in transcription and replication. Although prior studies have demonstrated that G4s exhibit higher mutation rates than canonical DNA, little is known about the substitution patterns and selection acting specifically on G4 stems and loops. In this study, we utilized Telomere-to-Telomere (T2T) genome assemblies from human and two non-human great apes (chimpanzee and Bornean orangutan) to analyze substitution spectra and selective constraints within G4s, focusing on differences between stems and loops. We observed that fixed nucleotide substitutions leading to the gain or loss of G4 structures are more frequently located at stems, while those in G4s conserved across species are more often found at loops. On the other hand, single nucleotide polymorphisms had similar frequencies between stems and loops. To evaluate selection, we employed two approaches: we computed the ratio of substitution to polymorphism frequencies at stems vs. loops and performed phylogenetic modeling using PhyloFit. Both methods consistently revealed that stems of shared G4s experience stronger purifying selection than loops, particularly at promoters, enhancers, and UTRs. Our results provide novel insights into the sequence variation and selection of G4s, informing our understanding of their contributions to genome evolution and function."
abstract_short = ""

# Is this a selected publication? (true/false)
selected = false

# Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["mitochondrial_mutation_dynamics"]


# Slides (optional).
#   Associate this page with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides = "example-slides"` references 
#   `content/slides/example-slides.md`.
#   Otherwise, set `slides = ""`.
slides = ""

# Tags (optional).
#   Set `tags = []` for no tags, or use the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []

# Links (optional).
url_pdf = ""
url_preprint = ""
url_code = ""
url_dataset = ""
url_project = ""
url_slides = ""
url_video = ""
url_poster = ""
url_source = ""

# Custom links (optional).
#   Uncomment line below to enable. For multiple links, use the form `[{...}, {...}, {...}]`.
# url_custom = [{name = "Custom Link", url = "http://example.org"}]


# Digital Object Identifier (DOI)
doi = ""



# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""
+++
