+++
title = "Complete genome sequences of six ape species unveiled"
subtitle = "Previously inaccessible regions reveal novel insights that may advance understanding of evolution and conservation genetics for endangered apes as well as human health"
date = 2025-04-09T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Christine Yu"]

# Tags and categories
# For example, use `tags = []` for no tags, or the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []
categories = []

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["sex_chromosome_evolution"]

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""


[header]
  image = ""

+++
{{< youtube hpxZwaQZqPc >}}

<br>
{{< figure
      class="image-center"
      src="ape.png"
      caption="Credit: guenterguni/Getty Images. All Rights Reserved."
>}}
UNIVERSITY PARK, Pa. — Differences among the DNA of seven ape species — including humans — are greater than originally thought, according to an international team led by researchers at Penn State, the National Human Genome Research Institute (NHGRI) and the University of Washington. They revealed the genetic details with “complete” reference genomes, which are standardized sequences of a species’ genes and other chromosomal regions. Complete reference genomes allow for comparison between species, enabling researchers to look for variations in DNA that might impact a species’ health and survival. Assembling full DNA sequences from one end of each chromosome to the other was previously not possible due to technological and algorithmic limitations.

The findings, published today (April 9) in [Nature](https://www.nature.com/articles/s41586-025-08816-3), shed light on primate evolution and highlight new species-specific genes and genes with multiple copies.

"This work uncovered novel adaptive signatures in genes related to diet, immune response and cellular activity, offering precise insights into the evolutionary pressures shaping great ape genomes,” said [Christian Huber](https://science.psu.edu/bio/people/cdh5313), assistant professor of biology at Penn State and co-author of the study.

The sequences are more accurate and complete than previous reference genomes and offer greater insights into genetic functions and disease mechanisms, including pinpointing genes and variants that are significant to health. Studies based on the new reference genomes, the team said, could help advance conservation genetics for endangered species as well as understanding of human evolution and health.

“This is a milestone for comparative genomic studies that allows us to appreciate the evolution of the genome in its full detail and complexity, which we couldn’t do before because we were working with incomplete genomes,” said [Kateryna Makova](https://science.psu.edu/bio/people/kdm16), Verne M. Willaman Chair of Life Sciences, professor of biology at Penn State and co-senior author of the study. “This work should serve as a definitive baseline for future evolutionary studies of humans and our closest living ape relatives.”

The human genome was first sequenced in 2001, and since then, sequencing other ape genomes has been a goal, Makova said, because other ape genomes can help explain the evolutionary history of the base pairs that make up the human genome. Reference genomes can also help in efforts to protect endangered ape species. By understanding the genetic diversity of apes, scientists can formulate better conservation strategies to increase the population of these animals.

Here, using advanced sequencing techniques and computational algorithms, the team decoded the genome for each of six ape species — chimpanzee, bonobo, gorilla, Bornean orangutan, Sumatran orangutan and siamong — and compared them to each other, as well as to the human genome. The researchers applied long-read sequencing technologies to decipher the genome, which allowed the researchers to read long segments of DNA and assemble them together from one end of each chromosome to the other, without any gaps in sequence. As a result, the team uncovered new genes and multi-copy gene families that are specific to a species or a group of species.

“These new genes might partly be responsible for the differences we see between the species, including human-specific traits such as intelligence,” said Karol Pál, postdoctoral scholar at Penn State and co-author of the study.

The team also identified evolutionary markers, which will allow researchers to compare closely related species, and interpreted previously inaccessible regions of the genome. They found that these regions were rich in sequences that form non-canonical DNA. The researchers explained that non-canonical DNA are unusual structures that don’t follow the typical double-helix arrangement of DNA base pairs and can play a role in regulating critical cellular processes, such as gene transcription and DNA replication. Some of these structures have also been linked to diseases like cancer. Mapping these regions will allow researchers to study these structures further and potentially understand how they give rise to health or disease.

Previous versions of ape reference genomes were incomplete and fragmented because scientists were limited by experimental methods that made assembling complete genomes difficult. Makova also noted that previous computational algorithms were inadequate to analyze such complex sequencing information, slowing down the process.

“Imagine that you’re trying to assemble a 2,000-piece jigsaw puzzle without a picture of what it should look like. It’s hard to assemble in the correct order,” Makova said.

In recent years, DNA sequencing and assembly technology have advanced. Penn State’s collaborators from NHGRI also improved the computational algorithms to work much faster and with greater accuracy, requiring little manual intervention. The team used similar techniques to sequence the  [complete human Y chromosome] (https://www.psu.edu/news/research/story/dna-sequence-human-y-chromosome-fully-determined-first-time) and the [complete genomes of the two sex chromosomes of living great ape species] (https://www.psu.edu/news/eberly-college-science/story/complete-x-and-y-chromosome-sequences-living-great-ape-species), both published in Nature in the last two years.

Data from the reference genomes are available online to the public.

Moving forward, the team said they hope to expand this approach of high-quality sequencing and assembly to gather more data on individual apes within the species studied in this paper as well as study new species, such as gibbons. 

In addition to Makova, Huber and Pál, the research team at Penn State includes Yana Safonova, assistant professor of computer science and engineering; Abigail N. Sequeira, doctoral student in biology; Laura Carrel, professor of biochemistry and molecular biology; Robert S. Harris, assistant research professor of biology, now retired; Kaivan Kamali, computational scientist in the Departments of Biology and of Biochemistry and Molecular Biology, now chief data scientist at Cognizant; Barbara McGrath, associate research professor of biology, now retired; Zachary Szpiech, assistant professor of biology; Saswat K. Mohanty, graduate student in biology; Linnéa Smeds, postdoctoral scholar in biology; and Edmundo Torres-González, graduate student in biology.

The team was co-led by Makova and co-corresponding study authors Adam Phillippy, senior investigator at NHGRI, and Evan Eichler, professor of Genome Sciences at the University of Washington. A full list of authors for this paper is available here.

Funding from the National Institutes of Health and the Penn State Eberly College of Science supported the work by Penn State researchers on this study.




