+++
title = "Complete X and Y chromosome sequences of living great ape species determined"
subtitle = "New end-to-end sequences uncover enormous variation on the Y chromosome, informing human evolution and disease as well as conservation genetics of endangered apes"
date = 2024-05-29T00:00:00
draft = false

# Authors. Comma separated list, e.g. `["Bob Smith", "David Jones"]`.
authors = ["Penn State Eberly College of Science"]

# Tags and categories
# For example, use `tags = []` for no tags, or the form `tags = ["A Tag", "Another Tag"]` for one or more tags.
tags = []
categories = []

# Projects (optional).
#   Associate this post with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `projects = ["deep-learning"]` references 
#   `content/project/deep-learning/index.md`.
#   Otherwise, set `projects = []`.
projects = ["sex_chromosome_evolution"]

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
[image]
  # Caption (optional)
  caption = ""

  # Focal point (optional)
  # Options: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight
  focal_point = ""


[header]
  image = ""

+++

Newly generated, complete “end-to-end” reference genomes for the sex chromosomes of five great ape species and one lesser ape species — produced by an international collaborative team led by researchers at Penn State, the National Human Genome Research Institute and the University of Washington — highlight extremely rapid changes on the male-specific Y chromosome among ape species. These findings shed light on the evolution of sex chromosomes and inform understanding of diseases related to genes on these chromosomes in both apes and humans.

“The Y chromosome is important for human fertility, and the X chromosome harbors genes critical for reproduction, cognition and immunity,” said Kateryna Makova, Verne M. Willaman Chair of Life Sciences, professor of biology at Penn State and leader of the research team. “Our study opens doors for many future investigations of sex chromosomes, how they evolved, and diseases associated with them. The living non-human great ape species we studied are all endangered. The availability of their complete sex chromosome sequences will facilitate studies of their sex-specific dispersal in the wild and of their genes important for reproduction and fertility.”

Such reference genomes act as a representative example that are useful for future studies of these species. The team found that, compared to the X chromosome, the Y chromosome varies greatly across ape species and harbors many species-specific sequences. However, it is still subject to purifying natural selection — an evolutionary force that protects its genetic information by removing harmful mutations.

The new study appears May 29 in the journal [Nature](https://www.nature.com/articles/s41586-024-07473-2).

"Researchers sequenced the human genome in 2001, but it wasn’t actually complete,” Makova said. “The technology available at the time meant that certain gaps weren’t filled in until a renewed effort led by the Telomere-to-Telomere, or T2T, Consortium in 2022-23. We leveraged the experimental and computational methods developed by the [Human T2T Consortium](https://www.psu.edu/news/research/story/dna-sequence-human-y-chromosome-fully-determined-first-time/) to determine the complete sequences for the sex chromosomes of our closest living relatives—great apes.”

The team produced complete sex chromosome sequences for five species of great apes — chimpanzee, bonobo, gorilla, Bornean orangutan and Sumatran orangutan, which comprise most great ape species living today — as well as a lesser ape, siamang. They generated sequences for one individual of each species. The resulting reference genomes act as a map of genes and other chromosomal regions, which can help researchers sequence and assemble the genomes of other individuals of that species. Previous sex chromosome sequences for these species were incomplete or — for the Bornean orangutan and siamang — did not exist.

“The Y chromosome has been challenging to sequence because it contains many repetitive regions, and, because traditional short-read sequencing technology decodes sequences in short bursts, it is difficult to put the resulting segments in the correct order,” said Karol Pál, postdoctoral researcher at Penn State and a co-first author of the study. “T2T methods use long-read sequencing technologies that overcome this challenge. Combined with advances in computational analysis, on which we collaborated with Adam Phillippy’s group at the NHGRI, this allowed us to completely resolve repetitive regions that were previously difficult to sequence and assemble. By comparing the X and Y chromosomes to each other and among species, including to the previously generated human T2T sequences of the X and the Y, we learned many new things about their evolution.”

{{< figure
      class="image-center"
      src="apexy.jpeg"
      caption="Newly generated, complete genomes for the sex chromosomes of six primate species — produced by an international collaboration led by researchers at Penn State and the National Human Genome Research Institute — reveal rapid evolution on the Y chromosome among apes. These results may inform conservation of these endangered species and shed light on sex-related genetic diseases in both humans and our closest living relatives. **Credit: Design: Bob Harris; Photography: San Diego Zoo and Tulsa Zoo.**"
>}}

**High variability on the Y chromosome**
“Sex chromosomes started like any other chromosome pair, but the Y has been unique in accumulating many deletions, other mutations and repetitive elements because it does not exchange genetic information with other chromosomes over most of its length,” said Makova, who is also the director of the [Center for Medical Genomics](https://cmg.psu.edu/) at Penn State.

As a result, across the six ape species, the research team found that the Y chromosome was much more variable than the X over a variety of characteristics, including size. Among the studied apes, the X chromosome ranges in size from 154 million letters of the ACTG alphabet — representing the nucleotides that make up DNA — in chimpanzee and human to 178 million letters in gorilla. In contrast, the Y chromosome ranges from 30 million DNA letters in siamang to 68 million letters in Sumatran orangutan.

The amount of DNA sequence shared between species was also more variable on the Y. For example, about 98% of the X chromosome aligns between human and chimpanzee, but only about a third of the Y aligns between them. The researchers found that this is in part because the Y chromosome is more likely to be rearranged or have portions of its genetic material duplicated.

Additionally, the percentage of the chromosome occupied by sequences that are repeated is highly variable on the Y. Whereas, depending on the species, 62% to 66% of the X chromosomes are occupied by repetitive elements, 71% to 85% of the Y chromosomes are occupied by them. These percentages are higher on both the X and the Y than in other chromosomes in the human genome.

**How the Y has survived**
“We found the ape Y to be shrinking, accumulating many mutations and repeats, and losing genes,” Makova said. “So why hasn’t the Y chromosome disappeared, as some previous hypotheses suggested? In collaboration with Sergei Kosakovsky Pond from Temple University and others, we found that the Y chromosome still has a number of genes evolving under purifying selection — a type of natural selection that keeps gene sequences intact. Many of these genes are important for spermatogenesis. This means that the Y chromosome is unlikely to disappear any time soon.”

The researchers found that many genes on the Y chromosome seem to use two strategies to survive. The first takes advantage of genetic redundancy — the presence of multiple copies of the same gene on a chromosome — so that intact copies of the gene can compensate for copies that might acquire mutations. The team quantified this genetic redundancy by completing the landscape of multi-copy gene families on ape sex chromosomes for the first time.

The second survival strategy takes advantage of palindromes, where the sequence of letters in the DNA alphabet is followed by the same, but inverted sequence, for example, ACTG-GTCA. When located within a palindrome, genes benefit from the palindrome’s ability to correct mutations.

“We found that the Y chromosome can exchange genetic information with itself between the repeated sequences of the two palindrome arms, which fold so that the inverted sequences align,” Pál said. “When two copies of the same gene are located within palindromes, and one copy is hit by a mutation, the mutation can be rescued by the genetic exchange with another copy. This can compensate for the Y’s lack of genetic information exchange with the other chromosomes.”

The research team obtained the complete sequences of palindromes on ape sex chromosomes also for the first time, as they were previously difficult to sequence and study. They found that palindromes are particularly abundant and long on the ape Y chromosome, yet they are usually only shared among closely related species. 

In collaboration with Michael Schatz and his team at Johns Hopkins University, the researchers also studied the sex chromosomes of 129 individual gorillas and chimpanzees to better understand the genetic variation within each species and search for evidence of natural selection and other evolutionary forces acting on them.

“We obtained substantial new information from previously studied gorilla and chimpanzee individuals by aligning their sex chromosome sequencing reads to our new reference sequences,” said Zachary Szpiech, assistant professor of biology at Penn State and an author of the paper. “While increasing the sample size in the future will be very helpful to improve our ability to detect signatures of different evolutionary forces, this can be ethically and logistically challenging when working with endangered species, so it is critical that we can get the most out of the data we do have.”

The researchers explored a variety of factors that could explain variation on the Y chromosome within gorillas and within chimpanzees, and this analysis revealed additional signatures of purifying selection on the Y. This confirms the role of this type of natural selection on the Y, as was discovered in their previous analyses of genes.

“The powerful combination of bioinformatic techniques and evolutionary analyses that we used allows us to better explain the evolutionary processes acting on sex chromosomes in our closest living relatives, great apes,” said Christian Huber, assistant professor of biology at Penn State and an author of the paper. “Additionally, the reference genomes we produced will be instrumental for future studies of primate evolution and human diseases.”

In addition to Makova, Pál, Szpiech and Huber, the research team at Penn State includes Kaivan Kamali, computational scientist in the departments of biology and of biochemistry and molecular biology; Troy LaPolice, graduate student in bioinformatics and genomics; Paul Medvedev, professor of computer science and engineering and of biochemistry and molecular biology; Sweetalana, research assistant in the department of biology; Huiqing Zeng, research technologist in biology; Xinru Zhang, graduate student in bioinformatics and genomics; Robert Harris, assistant research professor of biology, now retired; Barbara McGrath, associate research professor of biology, now retired; and Sarah Craig, associate research professor of biology, currently a program officer at the National Institutes of Health. The co-authors also included Penn State alumni Monika Cechova, currently a postdoctoral fellow at the University of California Santa Cruz, and Melissa Wilson, currently an associate professor at Arizona State University.

In addition to Makova, the team was co-led by co-corresponding study authors Adam Phillippy, senior investigator at NHGRI, and Evan Eichler, professor of Genome Sciences at the University of Washington. A full list of authors for this paper is available here.

Funding from the National Institutes of Health supported this research.



